document-generator-interface/pom.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.companieshouse</groupId>
+    <artifactId>document-generator</artifactId>
+    <version>unversioned</version>
+  </parent>
+  <groupId>uk.gov.companieshouse</groupId>
+  <artifactId>document-generator-interface</artifactId>
+  <version>unversioned</version>
+  <name>document-generator-interface</name>
+  <url>http://maven.apache.org</url>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
```

Comment:
Can you add spacing to make this easier to read please? This will get difficult to read as it gets longer.

Reply:
I agree, please make it 4 spaces instead of 2!